packages/edit-site/src/components/global-styles/font-library-modal/local-fonts.js

```diff
@@ -150,29 +156,39 @@ function LocalFonts() {
 		const response = await installFonts( fontFamilies );
 		const installNotice = getNoticeFromInstallResponse( response );
 		setNotice( installNotice );
+		setIsUploading( false );
 	};
 
 	return (
 		<>
 			<Spacer margin={ 16 } />
 			<DropZone onFilesDrop={ handleDropZone } />
 			<VStack className="font-library-modal__local-fonts">
-				<FormFileUpload
-					accept={ ALLOWED_FILE_EXTENSIONS.map(
-						( ext ) => `.${ ext }`
-					).join( ',' ) }
-					multiple={ true }
-					onChange={ onFilesUpload }
-					render={ ( { openFileDialog } ) => (
-						<Button
-							className="font-library-modal__upload-area"
-							onClick={ openFileDialog }
-						>
-							<span>{ __( 'Upload font' ) }</span>
-						</Button>
-					) }
-				/>
-				{ notice && (
+				{ ! isUploading && (
+					<FormFileUpload
+						accept={ ALLOWED_FILE_EXTENSIONS.map(
+							( ext ) => `.${ ext }`
+						).join( ',' ) }
+						multiple={ true }
+						onChange={ onFilesUpload }
+						render={ ( { openFileDialog } ) => (
+							<Button
+								className="font-library-modal__upload-area"
+								onClick={ openFileDialog }
+							>
+								<span>{ __( 'Upload font' ) }</span>
+							</Button>
+						) }
+					/>
+				) }
+				{ isUploading && (
+					<FlexItem>
+						<Spacer margin={ 32 } />
+						<ProgressBar className="font-library-modal__upload-area__progress-bar" />
+						<Spacer margin={ 32 } />
+					</FlexItem>
+				) }
```

Comment:
I think we can achieve a more consistent layout by re-using the `font-library-modal__upload-area` class, as this already has a fixed height. Something like this:

```suggestion
				{ isUploading && (
					<FlexItem>
						<div className="font-library-modal__upload-area">
							<ProgressBar />
						</div>
					</FlexItem>
				) }
```

We don't want the progress bar to have a grey background, so we could adjust the CSS so the background colour is specific to the upload button:

```
button.font-library-modal__upload-area {
	background-color: #f0f0f0;

}
```

This should create a more consistent layout between the not-uploading is isUploading states:

| Not-uploading | isUploading |
| -------------- | ------------ |
| <img width="977" alt="image" src="https://github.com/WordPress/gutenberg/assets/1645628/f5c133a5-04a5-4779-93c8-861319295d16"> | <img width="977" alt="image" src="https://github.com/WordPress/gutenberg/assets/1645628/47c52b0f-cb37-4418-ae0f-5e9a5409b2d7"> |

Reply:
Sure would be looking onto this.

Thanks.